.github/workflows/pr-title-update.yml

```diff
@@ -2,17 +2,17 @@ name: Update PR title
 
 on:
   pull_request_target:
-    types: [opened, edited]
+    types: [opened, edited, reopened]
     branches:
       - "release_**"
 
 jobs:
   update-title:
+    if: github.event.action != 'edited' || github.event.changes.base.ref.from != ''
     runs-on: ubuntu-latest
     permissions:
       pull-requests: write
     steps:
-      - uses: actions/checkout@v4
```

Comment:
This should not be removed.
It could get replaced by initiating a git repo as it is necessary to run the rest of the code.
This is the error code that it faces: `failed to run git: fatal: not a git repository (or any of the parent directories): .git`

Reply:
This could be fixed by adding this to the end of `gh pr edit`:
`--repo ${{ github.repository }}`

Reply:
> It would be better to add `${{ github.repository }}` as an env variable first and then use it in the script.

This is necessary only for variables that are under control of the user opening the PR, no?